config/file_config.go

```diff
@@ -350,6 +353,14 @@ func (f *fileConfig) validateGeneralConfigs() error {
 	default:
 		return fmt.Errorf("invalid CacheOverrunStrategy: '%s'", st)
 	}
+
+	stressReliefConfig := f.GetStressReliefConfig()
```

Comment:
Can you remove these changes and save them for the other PR?

Reply:
Oh, not sure how this happened. Accidental. Should be fixed now.